architecture/additional_concepts/storage.adoc

```diff
@@ -449,6 +449,15 @@ Fibre Channel and HostPath persistent volumes do not support mount options.
 ====
 endif::openshift-enterprise,openshift-origin[]
 
+=== Recursive chown
+
+Each time a persistent volume is mounted to a pod, or when a pod restart occurs, the volume's ownership and permissions are recursively changed to match those of the pod. Group read/write access is added to the volume when *chown* is performed on all files and directories in the PV. This enables the process running inside the pod to access the PV's file system.
```

Comment:
s/*chown*/`chown`/
_Use backticks for all non-GUI "system items", including Inline commands_ (https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference)

Reply:
Updated from *chown* to `chown`.